client/src/index.jsx

```diff
@@ -1,4 +1,5 @@
 import ReactDOM from 'react-dom';
 import App from './App.jsx';
+import './index.css';
```

Comment:
I think you need to use one style guide.

Reply:
i use it here to have global css variables and it is more convenient to use here app.css, not app.module.css. In this case all variables will be available in all components